### DISCUSSION TITLE
how to enable notifications for group and forum in groups?

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#148] [Installation]
Hello Team,
Hope you are doing well!..
We are using my groups menu on portal side bar to create groups and add members in it. But we are not getting notifications after adding user or removing group. DO we need any extra configuration or setup for enabling these notifications?
Also Do we have notification for forum also like for eg: starting new discussion/thread in forum or after adding comment in thread?? Do we notify users about this?

here are the versions we are using
1. Sunbird-Ed portal - 4.10.0_RC12
2. Notification-service - release-4.6.0_RC2
3. discussions-middleware - release-4.10.0_RC3

@coolbung, @gandham-santhosh, @VenkatKadiveti, @arunpilli21@sknirmalkar89, @manzarul, @aishwa8141, @vinukumar-vs

**Comment by Jayaprakash8887** [comment]
@vinukumar-vs Any pointers?

**Reply by Krishnaj20** [reply]
@Jayaprakash8887 - without proper logs and analysis it will be difficult for the Sunbird community to provide any insights on the issue faced. 
@paritshivani @coolbung - Pls share the required details.

@rajeevsathish @amitpriyadarshi
